src/qz/common/TrayManager.java

```diff
@@ -92,36 +95,33 @@ public TrayManager(boolean isHeadless) {
         shortcutCreator = DeployUtilities.getSystemShortcutCreator();
         shortcutCreator.setShortcutName(Constants.ABOUT_TITLE);
 
-        SystemUtilities.setSystemLookAndFeel();
+        // TODO: DON'T FORGET TO ADDRESS THIS. It causes a gtk2 vs 3 error on elementaryOS
+        // This at least mitigates the problem, still need to fix
+        if (!dorkbox.util.OS.isLinux() || dorkbox.systemTray.jna.linux.Gtk.isGtk2) {
```

Comment:
This could be done with our isLinux(). I am using the dorkbox one atm just to ensure that its behavior lines up with that of the systemtray itself.